src/autopas/selectors/AutoTuner.h

```diff
@@ -141,23 +148,30 @@ class AutoTuner {
    */
   template <class PairwiseFunctor>
   void addTimeMeasurement(PairwiseFunctor &pairwiseFunctor, long time) {
+    const auto &currentConfig = _tuningStrategy->getCurrentConfiguration();
+
     if (pairwiseFunctor.isRelevantForTuning()) {
       if (_samples.size() < _maxSamples) {
         AutoPasLog(trace, "Adding sample.");
         _samples.push_back(time);
         // if this was the last sample:
         if (_samples.size() == _maxSamples) {
           auto reducedValue = OptimumSelector::optimumValue(_samples, _selectorStrategy);
-          _tuningStrategy->addEvidence(reducedValue, _iteration);
+
+          _evidences[currentConfig].emplace_back(_iteration, reducedValue);
+          auto smoothedValue = smoothing::smoothLastPoint(_evidences[currentConfig], 5);
```

Comment:
so right now, 5 pieces of evidence are used for the smoothing and it is always enabled, correct?

Reply:
Up to 5, depending on how many are available but yes.
I actually see no advantage in disabling this feature. We could make the 5 a changeable parameter from the outside if you like but it is difficult to compare multiple runs anyway.